guides/source/caching_with_rails.md

```diff
@@ -517,16 +517,6 @@ To get started, add the redis gem to your Gemfile:
 gem 'redis'
 ```
 
-You can enable support for the faster [hiredis](https://github.com/redis/hiredis)
-connection library by additionally adding its ruby wrapper to your Gemfile:
-
-```ruby
-gem 'hiredis'
-```
-
-Redis cache store will automatically require and use hiredis if available. No further
-configuration is needed.
-
```

Comment:
I chose to remove this, because it won't work with Redis `5.0`, instead users would have to install `hiredis-client`. I'm not sure if it's worth documenting this here.